MacPass/MPPasswordInputController.m

```diff
@@ -130,4 +140,73 @@ - (void)_showError:(NSError *)error {
   self.errorInfoTextField.hidden = NO;
 }
 
+- (IBAction)showTouchIdDialog:(id)sender {
+	[self _enableTouchID];
+}
+
+- (void) appFocused {
```

Comment:
Please remove any unnecessary logging. I know it's nice to see that suff work ;-)